python/ray/tests/test_runtime_env_working_dir.py

```diff
@@ -21,10 +22,32 @@
 GS_PACKAGE_URI = "gs://public-runtime-env-test/test_module.zip"
 
 
-@pytest.mark.parametrize("option", [
-    "failure", "working_dir", "py_modules", "py_modules_path",
-    "working_dir_path"
-])
+@pytest.fixture(scope="function")
+def tmp_working_dir():
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        path = Path(tmp_dir)
+
+        hello_file = path / "hello"
+        with hello_file.open(mode="w") as f:
+            f.write("world")
+
+        module_path = path / "test_module"
+        module_path.mkdir(parents=True)
+
+        test_file = module_path / "test.py"
+        with test_file.open(mode="w") as f:
+            f.write("def one():\n")
+            f.write("    return 1\n")
+
+        init_file = module_path / "__init__.py"
+        with init_file.open(mode="w") as f:
+            f.write("from test_module.test import one\n")
+
+        yield tmp_dir
```

Comment:
Very sorry for not noticing this earlier--could you please remove this?  `tmp_working_dir` is already defined in `tests/conftest.py`, and we shouldn't duplicate it.

Reply:
No worries. Will do! Thanks